README.md

```diff
@@ -7,25 +7,25 @@ Mypy: Optional Static Typing for Python
 What is mypy?
 -------------
 
-Mypy is an optional static type checker for Python.  You can add type
+[Mypy](https://mypy-lang.org) is an optional static type checker for Python.  You can add type
```

Comment:
I agree it would be nice to have a link to that site in the README but I don't like doing it at the first word of the first sentence of the first paragraph of the first section. It feels like encouraging the user to follow a link before they've had a chance to start reading the current page. Maybe just add a sentence at the end of this section with "For more information blah blah blah?"